tstest/natlab/machost/README.md

```diff
@@ -0,0 +1,84 @@
+# macOS VM's for tstest and natlab
+
+## Building
+
+```
+%make all
+```
+
+Will build both the TailMac and the VMHost app.  You will need a developer account.  The default bundle identifiers
+default to tailscale owned ids, so if you don't have (or aren't using) a tailscale dev account, you will need to change this.
+This should build automatically as long as you have a valid developer cert.  Signing is automatic.  The binaries both
+require proper entitlements, so they do need to be signed.
+
+There are separate recipes in the makefile to rebuild the individual components if needed.
+
+All binaries are copied to the bin directory.
+
+You can generally do all interactions via the TailMac command line util.
+
+## Locations
+
+Everything is persisted at ~/VM.bundle
+
+Each vm gets it's own directory under there.  
+
+RestoreImage.ipsw is used to build new VMs.  You may replace this manually if you wish.
+
+Individual parameters for each instance are saved in a json config file (config.json)
+
+## Installing
+
+### Default a parameters
+
+The default virtio socket device port is 51009
+The default server socket for the virtual network device is /tmp/qemu.sock
```

Comment:
maybe we should default to a different filename for the different protocol, then cmd/vnet could listen on both in both styles and work with either/both clients.

So like `/tmp/qemu-dgram.sock` or something.